### ISSUE TITLE
Hello, the ground truth images in the data set are all in grayscale mode, and the mask cannot be greater than 128

### ISSUE THREAD

**Issue by moliniao** [SysCV/sam-hq#42]
Hello, I would like to ask that the ground truth images in the dataset are all in grayscale mode, and the mask cannot be greater than 128. Is the masks[b_i]>128 in masks_to_boxes and masks_sample_points still effective?

**Comment by lkeab** [comment]
is this your own training dataset?

**Comment by moliniao** [comment]
I use the author's dataset cascade_psp

**Comment by ymq2017** [comment]
Hi, GT masks are in grayscale mode with a value range of 0-255. The white region has a value of 255.

**Comment by sym330** [comment]
可以改为>0吧，我是这样改的
